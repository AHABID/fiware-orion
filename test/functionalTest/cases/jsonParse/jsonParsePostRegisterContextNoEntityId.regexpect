HTTP/1.1 200 OK
Content-Length: 211
Content-Type: application/json
Date: REGEX(.*)

{
    "errorCode": {
        "code": "400",
        "details": "JSON Parse Error: <unspecified file>(1): expected object name",
        "reasonPhrase": "Bad Request"
    },
    "registrationId": "000000000000000000000000"
}
++++++++++++++++++++
