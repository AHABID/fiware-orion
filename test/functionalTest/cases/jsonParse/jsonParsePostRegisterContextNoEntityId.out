HTTP/1.1 200 OK
Content-Length: 211
Content-Type: application/json
Date: Mon, 02 Jun 2014 08:19:23 GMT

{
    "errorCode": {
        "code": "400", 
        "details": "JSON Parse Error: <unspecified file>(1): expected object name", 
        "reasonPhrase": "Bad Request"
    }, 
    "registrationId": "000000000000000000000000"
}
++++++++++++++++++++
